lib/src/client.dart

```diff
@@ -56,6 +57,69 @@ class EOSClient {
     });
   }
 
+  /// Get table rows (eosio get table ...)
+  Future<List<dynamic>> getTableRows(
```

Comment:
try not use `dynamic`. The project intents to use typed structure

Reply:
but the table structures are different, do you have a good idea?

Reply:
yeah, using Map to represent single json object makes sense.

Reply:
when I change `dynamic` to `Map<String, dynamic>`, I get the error:

`type 'List<dynamic>' is not a subtype of type 'FutureOr<List<Map<String, dynamic>>>'`

I can cast to `Map<String, dynamic>` in a loop, but it will spend some time.